docs/api-reference/mapbox/overview.md

```diff
@@ -135,6 +135,8 @@ export class App extends React.Component {
         initialViewState={INITIAL_VIEW_STATE}
         controller={true}
         onWebGLInitialized={this._onWebGLInitialized}
+        // To render vector tile polygons correctly
```

Comment:
Same issue here.

I would prefer to avoid duplicating examples in multiple documentations. It's very hard to keep track of which ones to update when the API is changed. You can link to another doc instead.

Reply:
Ok, I removed the example in `mapbox-layer.md`